api/src/json-parser.ts

```diff
@@ -0,0 +1,31 @@
+import { bytes } from "@typeberry/block";
+import { isBigIntString } from "./number-utils";
+
+export function parse(json: string) {
+  return JSON.parse(json, (_key, value) => {
+    if (typeof value === "string" && value.startsWith("0x")) {
+      return bytes.Bytes.parseBytes(value, value.length / 2 - 1);
+    }
+
+    if (typeof value === "string" && isBigIntString(value)) {
+      return BigInt(value);
```

Comment:
Can we please replace the parser with https://github.com/sidorares/json-bigint so that we don't have to deal with the stringified bigints?

Reply:
unfortunately we can't use this particular parser because we still have to deal with byte blobs and the parser you proposed doesn't allow to define reviver/replacer functions. I checked a few different libs and they have the same problem. if you don't mind I would replace the parser as a separate PR

Reply:
fixed using this package: https://www.npmjs.com/package/json-bigint-patch